### ISSUE TITLE
add set_tile(position: Vector2, tile: int) method for TileMap

### ISSUE THREAD

**Issue by alexzheng** [godotengine/godot-proposals#1108]
**Describe the project you are working on:**

**Describe the problem or limitation you are having in your project:**
When just change a tile without modify other options, the code maybe:
tilemap.set_cellv(cell, new_tile, tilemap.is_cell_x_flipped(cell.x, cell.y), tilemap.is_cell_y_flipped(cell.x, cell.y), tilemap.is_cell_transposed(cell.x, cell.y))  

it's really a long call.

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**
can we have a method just set the tile, keep other options just as it is

tilemap.set_tile(cell, new_tile)

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**

**Is there a reason why this should be core and not an add-on in the asset library?:**
It's just a handy call.

**Comment by aaronfranke** [comment]
The booleans are optional parameters, so you can already do this if you want all of the booleans to be false: `tilemap.set_cellv(cell, new_tile)`

**Comment by erictuvesson** [comment]
When using `tilemap.set_cellv(cell, new_tile)` you will reset the tile transform.

I have read that the plan is to remove the transform from the tilemap and move it over to the tileset instead.

@groud maybe you have some more input here?

**Comment by groud** [comment]
> I have read that the plan is to remove the transform from the tilemap and move it over to the tileset instead.

Yes, that's what I had in mind. We will probably allow "tiles variations" directly in the tileset.
